src/server/scripts/Spells/spell_item.cpp

```diff
@@ -907,6 +907,48 @@ class spell_item_strong_anti_venom : public SpellScriptLoader
     }
 };
 
+class spell_item_anti_venom : public SpellScriptLoader
+{
+public:
+    spell_item_anti_venom() : SpellScriptLoader("spell_item_anti_venom") {}
+
+    class spell_item_anti_venom_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_item_anti_venom_SpellScript);
+
+        void HandleDummy(SpellEffIndex effIndex)
+        {
+            PreventHitDefaultEffect(effIndex);
+            if (Unit* target = GetHitUnit())
+            {
+                std::list<uint32> removeList;
+                Unit::AuraMap const& auras = target->GetOwnedAuras();
+                for (Unit::AuraMap::const_iterator itr = auras.begin(); itr != auras.end(); ++itr)
+                {
+                    Aura* aura = itr->second;
+                    if (aura->GetSpellInfo()->SpellLevel > 25 || aura->GetSpellInfo()->Dispel != DISPEL_POISON)
+                        continue;
```

Comment:
"Target is cured of poisons up to level 25." This makes the logic sound like, if this poison is level 25 and below, remove it. You made your logic sound like, if this aura is higher than level 25 and is a dispellable poison, continue checking other auras. It may be nitpicky but the first version sounds a bit more consistent so I would change it to something like this:

`if (aura->GetSpellInfo()->SpellLevel <= 25 && aura->GetSpellInfo()->Dispel == DISPEL_POISON)
{
removeList.push_back(aura->GetId());
}`

Also I'm quoting something **Shin** sayed on discord:
"_Avoiding brackets in one-liner if  or else blocks is a practice that historically has led to bugs (including in AC). This is why we should always add brackets to our if/else statements, despite our old codebase inherited from mangos/trinitycore does not use them. Let's try to add them whenever we add/modify code._"

Reply:
fixed the brackets